src/App.vue

```diff
@@ -182,6 +192,9 @@ const modelFiles = computed<Resource[]>(() => {
   return sortHelper(files, [{ name: unref(sortBy) }], unref(sortBy), unref(sortDir))
 })
 const activeModelFile = computed(() => unref(modelFiles)[unref(activeIndex)])
+const pageTitle = computed(() => `Preview for ${unref(activeModelFile)?.name}`)
+const fileId = computed(() => unref(currentFileContext).itemId)
```

Comment:
this should be left over from conflict resolution. these are not required anymore
```suggestion
```